client/app/redux/reducers/UserInformationReducers.js

```diff
@@ -0,0 +1,16 @@
+import createReducer from '../lib/createReducer';
+import * as types from '../actions/types';
+
+const initialUserInformationState = {
+    notificationID: undefined
+}
+
+export const userInformation = createReducer(initialUserInformationState, {
+    [types.SAVE_USER_NOTIFICATION_ID](state, action) {
+        const expoToken = action.payload.notificationID.split("[")[1].split("]")[0];
```

Comment:
Front end shouldn't have to worry about getting rid of the non-string token characters

Reply:
It does, since `registerForPushNotificationsAsync` returns the full string, though.

Reply:
Ah! Good note, yes, the backend doesn't do this.